zuul.d/jobs.yaml

```diff
@@ -10,3 +10,29 @@
         CEPH_DATASIZE: "2Gi"
       cifmw_kuttl_tests_operator_list:
         - manila
+
+- job:
+    name: manila-operator-tempest
+    parent: cifmw-base-multinode-tempest
+    attempts: 1
+    vars:
+      cifmw_edpm_prepare_update_os_containers: true
+      cifmw_set_openstack_containers_edpm: true
+      cifmw_set_openstack_containers_registry: "{{ content_provider_ip | default('nowhere') | trim }}"
```

Comment:
So I definitely think there's something weird happening here.
If you look at [1]:

```
NAME                               READY   STATUS                  RESTARTS   AGE
pod/ceph                           1/1     Running                 0          41m
pod/dnsmasq-dns-665cc5dc88-5bflr   0/1     Init:ImagePullBackOff   0          31m
pod/dnsmasq-dns-c67bc7d76-9szvx    0/1     Init:ImagePullBackOff   0          31m
pod/memcached-0                    0/1     ImagePullBackOff        0          31m
pod/openstack-cell1-galera-0       0/1     Init:ImagePullBackOff   0          31m
pod/openstack-galera-0             0/1     Init:ImagePullBackOff   0          31m
pod/ovn-controller-p7h4w           0/3     ImagePullBackOff        0          31m
pod/ovsdbserver-nb-0               0/1     ImagePullBackOff        0          31m
pod/ovsdbserver-sb-0               0/1     ImagePullBackOff        0          31m
pod/rabbitmq-cell1-server-0        0/1     Init:ImagePullBackOff   0          31m
pod/rabbitmq-server-0              0/1     Init:ImagePullBackOff   0          31m
pod/swift-ring-rebalance-rcq2j     0/1     ImagePullBackOff        0          31m
pod/swift-storage-0                0/16    ContainerCreating       0          31m
```
most of the containers are failing (see [2] for more details), and this happens due to the fact that the `content_provider_id` is not there, and `nowhere`, the default, is selected [2].

@viroel @son-vyas wdyt about this? I see the `content-provider` job being green [3] but I'm not sure why the ip address is not resolved.

[1] https://logserver.rdoproject.org/96/196/b47b03e5c2450cbbeee34b690a84f4b741826205/github-check/manila-operator-tempest/e2c3ef2/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/all_resources.log

[2] https://logserver.rdoproject.org/96/196/b47b03e5c2450cbbeee34b690a84f4b741826205/github-check/manila-operator-tempest/e2c3ef2/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/events.log

[3] https://review.rdoproject.org/zuul/buildset/ae028675f5264536aeee19a9a63eac87

Reply:
but from what I'm seeing we try to pull down containers from :80 instead of using :5001